wiki/Welcome/vi.md

```diff
@@ -0,0 +1,47 @@
+# Chào mừng
+
+Chào mừng đên osu!, một trò chơi nhịp điệu được lập trình bởi peppy với 4 chế độ chơi: osu!, chế độ bấm vòng tròn; osu!taiko, chế độ đánh trống; osu!catch, chế độ hứng quả; và osu!mania, chế độ bấm phím.
```

Comment:
```suggestion
Chào mừng đến osu!, một trò chơi nhịp điệu được phát triển bởi peppy với 4 chế độ chơi: osu!, chế độ bấm vòng tròn; osu!taiko, chế độ đánh trống; osu!catch, chế độ hứng quả; và osu!mania, chế độ bấm phím.
```
`developed` means a lot, from designing, coding, testing, maintaining, ...
Not just the `coding` (`lập trình`) part.
And I hope someone can suggest a good word for `key-smashing synthesizer`, shortening everything to `chế độ *` sounds very boring.
`chế độ bấm phím` means `key-hitting mode`, which doesn't convey the meaning properly.

Reply:
I'm thinking maybe `chế độ đập đàn` sounds better but it still stuck with the `chế độ *`